### DISCUSSION TITLE
Ability to have non employee users

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#5814] [Design]
There is a valid use case for non-employee users.
- software entities
- robots
- subcontractors

since automated entities may need to be able to log in to do their tasks

**Comment by ylavoie** [comment]
Accountant firms are another good example thereof.
Depending of their role, various access privileges are required.

**Comment by einhverfr** [comment]
https://sourceforge.net/p/ledger-smb/feature-requests/268/ is being consolidated into this ticket.
